src/Symfony/Bundle/FrameworkBundle/Controller/Controller.php

```diff
@@ -30,6 +28,11 @@ abstract class Controller implements ContainerAwareInterface
     use ContainerAwareTrait;
     use ControllerTrait;
 
+    public function __construct()
+    {
+        @trigger_error(sprintf('The "%s" class is deprecated since Symfony 4.2, use %s instead.', self::class, AbstractController::class), E_USER_DEPRECATED);
```

Comment:
One of the replacements is in quotes, the other not. Should be consistent with quotes

Reply:
Yes, sure. It seems like there was no quotes in the original file. Added quotes similarly to other cases.